examples/quotes-api/cmd/quotes-api/main.go

```diff
@@ -7,7 +7,7 @@ func main() {
 	}
 	defer cleanup()
 
-	api.server.GET("/", api.quoteController.GetQuote)
+	// api.server.GET("/", api.quoteController.GetQuote)
```

Comment:
cant we just remove this?

Reply:
I've split quotes-api `/cmd` folder into two different APIs:
```sh
examples/quotes-api/
├── cmd
│   ├── quotes-graphql-api
│   │   ├── config.go
│   │   ├── main.go
│   │   ├── wire.go
│   │   └── wire_gen.go
│   └── quotes-rest-api
│       ├── config.go
│       ├── main.go
│       ├── wire.go
│       └── wire_gen.go
```